app/src/androidTest/java/org/mozilla/fenix/helpers/TestHelper.kt

```diff
@@ -117,4 +125,20 @@ object TestHelper {
             }
         }
     }
+
+    fun setNetworkConnection(status: String) {
```

Comment:
nit: it'd be safer to use a boolean or enum

Reply:
done